hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -143,11 +143,7 @@ import Prelude qualified
 
 spec :: Spec
 spec = parallel $ do
-  describe "ChainState" $
-    roundtripAndGoldenSpecs (Proxy @ChainState)
```

Comment:
Why did this one get deleted?

Reply:
+1

Reply:
Because its covered by the added `"persisted event format"` test now. The commit message is not very clear on this (https://github.com/cardano-scaling/hydra/pull/1554/commits/01b24e382a0a10ae707fd17a5d49751db115f0bd) - do you want me to update it with more detail on the rationale so we would know later?